frontend/packages/dev-console/src/components/pipelineruns/PipelineRunsResourceList.tsx

```diff
@@ -4,6 +4,7 @@ import { referenceForModel } from '@console/internal/module/k8s';
 import { PipelineRunModel } from '../../models';
 import { runFilters } from '../pipelines/detail-page-tabs/PipelineRuns';
 import PipelineRunsList from './list-page/PipelineRunList';
+import { getBadgeFromType } from '@console/shared';
```

Comment:
I'd put this at the top of the import list, but otherwise this looks good.

Reply:
Moved import to the top